spec/serializers/disability_claim_detail_serializer_spec.rb

```diff
@@ -10,4 +10,8 @@
   it 'should include id' do
     expect(data['id']).to eq(disability_claim.id.to_s)
   end
+
+  it 'should not include raw HTML' do
+    expect(attributes.to_json.to_s.include?('<')).to be(false)
```

Comment:
I don't think there is HTML in the factory data by default. It would be good to do that specifically for this